### PR TITLE
update RawCollectionsExample.java

### PR DESCRIPTION
**Pull request by SuperZ2017**
use static method instead deprecated method

### CODE REVIEW

**Comment by SuperZ2017** [comment]
@googlebot I signed it!

**Comment by eamonnmcmanus** [comment]
Thanks, this is very helpful.

File: extras/src/main/java/com/google/gson/extras/examples/rawcollections/RawCollectionsExample.java [hunk 1]
```diff
@@ -45,8 +45,7 @@ public static void main(String[] args) {
     collection.add(new Event("GREETINGS", "guest"));
     String json = gson.toJson(collection);
     System.out.println("Using Gson.toJson() on a raw collection: " + json);
-    JsonParser parser = new JsonParser();
```
Comment (ladykraken) [inline extras/src/main/java/com/google/gson/extras/examples/rawcollections/RawCollectionsExample.java:48]: extras/src/main/java/com/google/gson/extras/examples/rawcollections/RawCollectionsExample.java

**Review by ladykraken — COMMENTED** [review]
#1953

**Pull request merged**